records/generator.go

```diff
@@ -17,6 +17,12 @@ import (
 	"github.com/mesosphere/mesos-dns/records/labels"
 )
 
+const (
+	// containerIPTaskStatusLabel represents the key of the state.json task status label
+	// which holds the container IP.
+	containerIPTaskStatusLabel = "Docker.NetworkSettings.IPAddress"
```

Comment:
This isn't used anywhere else than the `containerIP` method. We should define it inside and get rid of the comment since its not necessary so close to where it's used.